### DISCUSSION TITLE
can't get it to start.

### DISCUSSION THREAD

**Discussion by pepijnmm** [Aim2339/GiveawayS-old#18] [General]
Tried to start it but it tries "node_modules/.bin/node sharding.js" which doesn't exist. i also tried starting with "node sharding.js" but that gives "Shard failed to spawn." is there a special step I have to do first?
I wanted to edit the code a bit as i wanted the possibility to have multiple roles that could join the giveaway.

**Comment by Aim2339** [comment]
Hi, can u try deleting the modules (package-lock.json) and reinstalling it with ``npm install`` 

After doing this try running it again and let me know thanks!

**Comment by Aim2339** [comment]
Also, I think there is a issue with the sharding..a lil buggy it seems..so if that's the case I recommend following the code before this commit https://github.com/Aim2339/GiveawayS/commit/f5a570146025df6f32b13222a5174a633528f4eb
